6.72 port to main branch

zecoxao opened the pull request:
should have all 6.72 offsets except the ptrace patch

Commit:
Removed ps4-sdk and ps4-payload-sdk and will be added with new

Commit:
Added 6.72 offsets and consuming payload sdk from Scene-Collective repo

Commit:
Commenting out second ptrace patch as offset for 6.72 need to be figured out.

Yasich217 commented:
Tested build on 6.72, `vm_map_insert` failed

![image](https://user-images.githubusercontent.com/6113868/89133458-ab6fef00-d524-11ea-9372-ad234d1f7c98.png)

zecoxao commented:
well, then i guess someone should remove this pr until it's ready to work.
i got notified of this port but i suppose nobody tested it. figures...

DeathRGH committed:
Fix for 2nd ptrace patch

GiantPluto committed:
Merge pull request #1 from DeathRGH/patch-1

Fix for 2nd ptrace patch

Commit:
Enable UART and line endings fixed in build.sh

Commit:
Fixed disable ASLR patch.

Though the offset and the patch I identified were correct, I overlooked the data type since I updated bunch of them together. May be an idiot or blind?

GiantPluto committed:
Update README.md

GiantPluto committed:
Update README.md

jogolden commented:
Pluto, for the future, never remove credits. Updating an offset is a very simple task and the credit ought to be given to the original person who created the patch using their own abilities.

I am keeping this as a PR for now as it will override the 5.05 branch, I need to work on a multi-target build system before merging.

lockroz commented:
so tbh im facing an issue with this payload when i try to use this payload on some new games and new patches like new update of spider man and days gone none of the debuggers will connect to games or it will make them crash idk whats happening but i look in more info's in future 
appreciate ur concern.

Jamesvergara12 reviewed (commented):
Hopefully
